NotifyTests/IntegrationTests/Assertions.cs

```diff
@@ -0,0 +1,87 @@
+using Notify.Client;
+using Notify.Models;
+using Notify.Models.Responses;
+using Notify.Exceptions;
+using System;
+using System.Collections.Generic;
+
+using NUnit.Framework;
+
+namespace Notify.IntegrationTests.Assertions
+{
+	public class NotifyAssertions
+	{
+		static public void AssertNotification(Notification notification)
+		{
+			Assert.IsNotNull(notification.type);
+			String notificationType = notification.type;
+			String[] allowedNotificationTypes = { "email", "sms", "letter" };
```

Comment:
This would be probably be better in a enum.

Reply:
I think this was already in place, not sure if we need to update it now as an enum